Cargo.toml

```diff
@@ -100,6 +100,7 @@ dirs = { version = "6.0.0" }
 normalize-path = { version = "0.2.1" }
 pico-args = "0.5.0"
 rayon = { version = "1.10.0" }
+regex = "1.11.1"
```

Comment:
This will increase binary size a lot. :(

Reply:
Try removing features related to unicode.

Reply:
@Boshen Done, the binary size doesn't increase too much now.